app/uk/gov/hmrc/cataloguefrontend/AuthController.scala

```diff
@@ -32,22 +32,21 @@ import views.html.sign_in
 import scala.concurrent.Future
 
 @Singleton
-class AuthController @Inject()(val messagesApi: MessagesApi, authService: AuthService, configuration: Configuration)
-    extends FrontendController
-    with I18nSupport {
+class AuthController @Inject()(
+  authService: AuthService,
+  configuration: Configuration,
+  mcc: MessagesControllerComponents
+) extends MessagesAbstractController(mcc) with FrontendController {
```

Comment:
alternative is to do:
```
...
  val controllerComponents: MessagesControllerComponents
) extends FrontendController {
```
We should find the best way here for teams to use as this will be one of the most common changes they'll need to do